app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,143 @@
+                  </span>
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="field is-horizontal">
+            <div class="field-label is-normal">
+              <label class="label">Contraseña</label>
+            </div>
+            <div class="field-body">
+              <div class="field">
+                <p class="control is-expanded has-icons-left">
+                  <%= f.password_field :password, autocomplete: "new-password", class: "input", placeholder: "Contraseña" %>
+                  <span class="icon is-small is-left">
+                    <i class="fas fa-lock" aria-hidden="true"></i>
+                  </span>
+                </p>
+                <% if @minimum_password_length %>
+                  <em class="help">(Mínimo <%= @minimum_password_length %> caracteres)</em>
+                <% end %>
+              </div>
+              <div class="field">
+                <p class="control is-expanded has-icons-left">
+                  <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "input", placeholder: "Confirmar contraseña" %>
+                  <span class="icon is-small is-left">
+                    <i class="fas fa-lock" aria-hidden="true"></i>
+                  </span>
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="field is-horizontal">
+            <div class="field-label is-normal">
+              <label class="label">Domicilio</label>
+            </div>
+            <div class="field-body">
+              <div class="field">
+                <div class="control">
+                  <%= f.text_area :address, class: "textarea", placeholder: "Domicilio completo" %>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="field is-horizontal">
+            <div class="field-label"></div>
+            <div class="field-body">
+              <div class="field is-expanded">
+                <div class="field has-addons">
+                  <p class="control">
+                    <a class="button is-static">
+                      +52
+                    </a>
+                  </p>
+                  <p class="control is-expanded">
+                    <%= f.text_field :mobile, class: "input", placeholder: "Num. de teléfono o celular" %>
+                  </p>
+                </div>
+                <p class="help">Ejemplo: 312 123 4567</p>
+              </div>
+            </div>
+          </div>
+          <div class="field is-horizontal">
+            <div class="field-label">
+              <!-- Left empty for spacing -->
+            </div>
+            <div class="field-body">
+              <div class="field">
+                <div class="control">
+                  <%= button_tag type: 'submit', form:"new_user", class: "button is-info", disable_with: 'Ingresando...' do %>
```

Comment:
If your button tag content is juts text, you can avoid using a block
See examples [here](https://apidock.com/rails/v5.2.3/ActionView/Helpers/FormTagHelper/button_tag)